### PR TITLE
Amina/ social_icons_text_update

### PR DESCRIPTION
**Pull request by amina-deriv**

### CODE REVIEW

File: src/sass/_common/common.scss [hunk 1]
```diff
@@ -772,7 +772,7 @@ li strong.margin-right {
         background-color: $COLOR_GRAY;
     }
     .name {
-        text-transform: capitalize;
+        text-transform: capitalize !important;
```
Comment (akmal-binary) [inline src/sass/_common/common.scss:775]: Hi, I don't think this change would fix the mentioned issue with translation. Can we check with translation team whether these social network names are translated?

Comment (amina-deriv) [reply src/sass/_common/common.scss:775]: Hi  @akmal-binary, We are not translating social icon text in our sites. On checking with the QAteam, the actual issue is that for few languages, the social icon text starts with a lowercase letter.

Comment (akmal-binary) [reply src/sass/_common/common.scss:775]: Alright, could you also mention this on the redmine card, because later it might cause confusions among QA team.

**Review by akmal-binary — DISMISSED** [review]

File: src/sass/_common/common.scss (hunk 1 above)
Comment (amina-deriv) [reply src/sass/_common/common.scss:775]: sure. Thank you.

Comment (ridho-binary) [reply src/sass/_common/common.scss:775]: thanks @akmal-binary  for headsup , i was actually confused , i thought we are going to translate the social media button 😅 .
@amina-deriv i will update the remd card description as totally miss-leading 😄

**Review by akmal-binary — APPROVED** [review]

**Pull request merged**